images/OmegaT pixels/contributor.txt

```diff
@@ -0,0 +1 @@
+Leonardo Kenji Itiama Fulukava
```

Comment:
Path `OmegaT pixels` has a space character. In order to decrease trouble, Could you mind replace space chr with `_` or '.'?